plugins/by-name/oil-git-status/default.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  ...
+}:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "oil-git-status";
+  packPathName = "oil-git-status.nvim";
+  package = "oil-git-status-nvim";
+
+  description = ''
+    Add Git Status to oil.nvim directory listings.\
+    Git status is added to the listing asynchronously after creating the oil directory\
+    listing so it won't slow oil down on big repositories. The plugin puts the status in two new sign columns\
+    the left being the status of the index, the right being the status of the working directory.
+    signcolumn needs to be configured in oil:
```

Might be clearer to break this up into paragraphs?

```suggestion
    Add Git Status to oil.nvim directory listings.

    Git status is added to the listing asynchronously after creating the oil
    directory listing so it won't slow oil down on big repositories.

    The plugin puts the status in two new sign columns, the left being the
    status of the index, the right being the status of the working directory.

    `win_options.signcolumn` must to be configured in `plugins.oil.settings`:
```

Also, I may have confused the point a little in https://github.com/nix-community/nixvim/pull/3292#discussion_r2078741604, but we don't _usually_ manually wrap lines with `\` unless there's a specific reason we want a manual linebreak without a paragraph break.

---

oh okay sounds good to me